core/shared/src/main/scala/monocle/std/These.scala

```diff
@@ -19,6 +20,16 @@ trait TheseOptics {
       case Right(b) => That(b)
     }
 
+  def theseLeft[A, B, C]: POptional[Ior[A, B], Ior[C, B], A, C] =
```

Comment:
would you mind to rename those `pTheseLeft` and `pTheseRight` and create monophormic aliases e.g.  `theseLeft[A, B] = pTheseLeft[A, B, A]`?
This is because monomoprhic optics have better type inference.

Reply:
Done!